### PR TITLE
Pod Pilot

### PR DESCRIPTION
**Pull request by TheBeoni**
j247zmq
Adds Pod Pilot jumpsuit. Equiped by defalut.

### CODE REVIEW

**Comment by fj45** [comment]
Can you get a pic of it ingame?

**Comment by TheBeoni** [comment]
sure

**Comment by TheBeoni** [comment]
Also added fix for floating dots around Brig Physicians jumpsuit.

**Comment by TheBeoni** [comment]
Needed to do minor fix.

**Comment by fj45** [comment]
I don't personally like the jumpsuit.

**Comment by Fox-McCloud** [comment]
I don't personally care for it either.

**Comment by Sparkyninja** [comment]
I personally always thought that the pod pilot should be getting a set of warmer clothes, fitting in with the bomber jacket they spawn with. General idea being that Space is cold, so they'd get warm clothing (And NOT be in a hardsuit every damned time they are in space)

**Comment by TheBeoni** [comment]
that jumpsuit is based on British air force. I just went from olive green to black jumpsuit and red suspenders/belts.

**Comment by Fox-McCloud** [comment]
@Sparkyninja 

Generally, jumpsuits shouldn't have a whole lot of special things attached to them; a minor stat increase in line with their department is fine, but giving them special features out and above a regular jumpsuit isn't generally a good thing.

File: code/modules/clothing/under/jobs/security.dm [hunk 1]
```diff
@@ -154,4 +154,15 @@
 	_color = "brig_phys"
 	permeability_coefficient = 0.50
 	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 10, rad = 0)
+	flags = ONESIZEFITSALL
+
+//PodPilot
+/obj/item/clothing/under/rank/security/podpilot
+	desc = "Suit for proud and brave pod pilot.Comes with stylish red belt. There is tag on chest with /Pod Pilot/ writen on it."
```
Comment (Nienhaus) [inline code/modules/clothing/under/jobs/security.dm]: pilot.Comes < pilot. Comes

**Comment by Nienhaus** [comment]
So the suspenders on the jumpsuit seems to be holding up his crotch less then just his pants like most suspenders, I'm guessing that's just for the style of it? The sprite looks fine but I wouldn't guess pod pilot. Maybe remove the suspenders, leave the pants, and make the polo looking top a slightly lighter gray?

**Comment by TheBeoni** [comment]
Ill change it

**Comment by Fox-McCloud** [comment]
Merge conflicts ahoy! Also, do you plan on making an updated version to demonstrate?

**Comment by TheBeoni** [comment]
I will change jumpsuit look asap i didnt have time since i was in a place with crappy internet connection.

**Comment by TheBeoni** [comment]
c431utl
Its pretty much ERT/Blueshield suit with a bit of recoloring.

**Comment by TheDZD** [comment]
Merge conflicts.

**Comment by TheBeoni** [comment]
where? what?

**Comment by Fox-McCloud** [comment]
@TheBeoni 

Going to fix the merge conflicts or want me to close this?

**Comment by TheBeoni** [comment]
No idea what is wrong with it...

**Comment by Krausus** [comment]
Github *should* tell you what files are conflicting. If I were to guess, I'd say it's probably uniform.dmi, since you modified it in #1405 since opening this PR. Git won't even attempt to merge binary files, so you'll need to re-add the uniform to the current version of the file.

**Comment by Fox-McCloud** [comment]
@TheBeoni 

Do you plan on fixing the merge conflicts or?

**Comment by Fox-McCloud** [comment]
Going to just close this; feel free to make a new pull request!

**Pull request closed without merging**